regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -107,6 +108,10 @@ var CommentReviewView = Backbone.View.extend({
     this.submitMode();
   },
 
+  back: function() {
+    window.history.back();
+  },
```

Comment:
If they open in new tab or navigated directly, clicking back wouldn't trigger anything. Which I guess is a behavior I'd like to avoid, but wondering where it could go. Or perhaps just check history to not display the Back to your comment link when it's in a fresh state?